doc/features/tuning-policies/README.md

```diff
@@ -157,16 +157,175 @@ unexpected error, invoked in the following situations:
     - When the contacted host replies with an error, such as `overloaded`, `isBootstrapping`, `serverError`, etc. In 
     this case, the error is instance of `ResponseError`
 
-The [operation info][OperationInfo], passed as a parameter to the retry policy methods, exposes the `query` and query 
-`options` as properties.
+A default and base retry policy is included, along with `IdempotenceAwareRetryPolicy` that considers query idempotence. The default implementation doesn't use `IdempotenceAwareRetryPolicy`, in order to use it you should wrap the default policy with `IdempotenceAwareRetryPolicy`:
+
+``` js
+ client = new cassandra.Client({
+     contactPoints: contactPoints.split(','),
+     keyspace: keyspace,
+     authProvider: authProvider,
+     policies: {
+         retry: new cassandra.policies.retry.IdempotenceAwareRetryPolicy(new cassandra.policies.retry.RetryPolicy())
+     }
+ });
+```
+
+You can also override the policy implementation by inherits the default policy:
+
+``` js
+var cassandra = require('cassandra-driver');
```

Comment:
Creating a custom retry policy is something that is not very common, I don't think its a good idea to include a complete code sample about it.

Reply:
The whole point of the documentation was to give more info on the retry policy, cause when I tried to use it the documentation is very missing, also the logic of how the default policy works.
Maybe instead of put here a complete code sample we can put a link to the default policy or add some documentation on how the default one works/ how to override it without the implementation.

What do you think?

Reply:
Overriding the retry policy is not a very common use case, so we have to worry about good API docs and a clear reference implementation, I think linking to it would be enough.

Reply:
With API docs and good understanding (policy behaviors) would be sufficient to implement a custom retry policy.